backend/database/models/incidents.py

```diff
@@ -1,69 +1,144 @@
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer)  # TODO: foreign key
+    victim_id = db.Column(db.Integer)  # TODO: foreign key
+
+
+class IncidentDescription(db.Model):
+    # (Note) Removed all the `description_*` prefixes.
+    id = db.Column(db.Integer, primary_key=True)  # description id
+    incident_id = db.Column(db.Integer)  # TODO: foreign key
+    text = db.Column(db.Text)
+    type = db.Column(db.Text)  # TODO: enum
+    # TODO: are there rules for this column other than text?
+    source = db.Column(db.Text)
+
 
+class IncidentAction(db.Model):
+    id = db.Column(db.Integer, primary_key=True)  # action id
+    incident_id = db.Column(db.Integer)  # TODO: foreign key
+    date = db.Column(db.DateTime)
+    action = db.Column(db.Text)  # TODO: Not sure what this is.
+    actor = db.Column(db.Text)  # TODO: Not sure what this is.
+    notes = db.Column(db.Text)
 
-class Victim_Weapon_Enum(enum.Enum):
-    Unknown = 1
-    Firearm = 2
-    Blade = 3
-    Blunt = 4
-    NoWeapon = 5
-    Other = 6
 
+class IncidentTag(db.Model):
+    # TODO: rename as IncidentTagXRef ?
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer)  # TODO: foreign key
+    tag_id = db.Column(db.Integer)  # TODO: foreign key
 
-class Victim_Action_Enum(enum.Enum):
-    Unknown = 1
-    Speaking = 2
-    NoAction = 3
-    Fleeing = 4
-    Approaching = 5
-    Attacking = 6
-    Other = 7
 
+class Tag(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    term = db.Column(db.Text)
 
-class Cause_Of_Death_Enum(enum.Enum):
-    Unknown = 1
-    BluntForce = 2
-    GunShot = 3
-    Choke = 4
-    Other = 5
 
+class IncidentUseOfForce(db.Model):
+    # TODO: rename to IncidentUseOfForceXRef ?
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer)  # TODO: foreign key
+    use_of_force_id = db.Column(db.Integer)  # TODO: foreign key
 
-class Gender_Enum(enum.Enum):
-    Unknown = 1
-    Male = 2
-    Female = 3
-    Transgender = 4
 
+class ResultOfStopAtIncident(db.Model):
+    # TODO: Rename to IncidentResultOfStopXRef?
+    # TODO:
+    #  Does this need to be an crossref table?
+    #  Is there a many-to-many relationship?
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer)  # TODO: foreign key
+    result_of_stop_id = db.Column(db.Integer)  # TODO: foreign key
 
-class Race_Enum(enum.Enum):
-    Unknown = 1
-    White = 2
-    Black_African_American = 3
-    American_Indian_Alaska_Native = 4
-    Asian = 5
-    Native_Hawaiian_Pacific_Islander = 6
 
+class ResultOfStop(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    result = db.Column(db.Text)
 
-class Status_Enum(enum.Enum):
-    Unknown = 1
-    Unharmed = 2
-    Injured = 3
-    Disabled = 4
-    Deceased = 5
 
+# TODO: not sure if the below model should be used.
 
 class Incidents(db.Model):
```

Comment:
This is the only model we have that's pluralized. File name is also pluralized
```suggestion
class Incident(db.Model):
```

Reply:
Yeah that's the old model, and yep the model class names should be singular. I might as well delete it, I guess?

Reply:
Oh I see that. Yeah might be a good idea to remove it just to avoid confusion, along with the incidents query in the database/queries folder?